### DISCUSSION TITLE
re-run promise when property changes

### DISCUSSION THREAD

**Discussion by mcfiredrill** [Exelord/ember-await#12] [Q&A]
I want to re-run the promise to fetch more data when a property on my component changes. Is there any way to do this? I was thinking to use the `did-update` modifier from [ember-render-modifiers](https://github.com/emberjs/ember-render-modifiers)...

**Comment by Exelord** [comment]
Hi :) yes it is possible. You can use https://exelord.gitbook.io/ember-await/api/yielded-properties#reload
